tests/test_read_only.py

```diff
@@ -138,16 +156,20 @@ def lines(bytes):
 
 def test_model_loading(
     container_success: Generator[docker.models.containers.Container],
-    mode: str,
+    mode_success: str,
 ):
     container = next(container_success)
     wait_for_container(container)
 
     result = container.exec_run("ls -A1 /workspace")
-    if mode == "legacy":
+    if mode_success == "legacy":
```

Comment:
I suggest move the `if` statement out of the test, and instead create different tests. Common code can be shared via fixtures if needed.

Reply:
I haven't worked with `@pytest.mark.parametrize` before, but it's a nice feature. Implemented in https://github.com/DSD-DBS/capella-dockerimages/pull/73/commits/bfb07308d8354bb098492960f511065d415654e5.